optuna_dashboard/app.py

```diff
@@ -204,11 +204,12 @@ def delete_study(study_id: int) -> BottleViewReturn:
     @handle_json_api_exception
     def get_study_detail(study_id: int) -> BottleViewReturn:
         response.content_type = "application/json"
+        ntrials_client = int(request.params["ntrials_client"])
```

Comment:
It will raise an exception if clients call the API without this query parameter. Please return all trials when given no query parameter.

Reply:
Please add unit tests for the following scenarios.

1. Call the API without query parameters.
1. Call the API with invalid query parameters (e.g. It should return BAD REQUEST when given a large trial number such as 1000 even though there are only 100 trials).
1. Call the API with valid query parameters.

Reply:
And could you rename query parameters to `before`? I think it is the common naming in the cursor-based pagination for the Web API.

Reply:
No problem. These have been resolved in the latest commit